<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>580</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>580</height>
    </rect>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>190</y>
      <width>431</width>
      <height>170</height>
     </rect>
    </property>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>73</width>
       <height>24</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{color:white;border:1px solid  rgb(46,78,126);border-radius:6px;
background-color: rgb(46,78,126);}
QToolButton:hover{background-color:rgb(66,98,146);}
QToolButton:disabled{background-color:#cecece;}</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="privateKeyLineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>361</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QToolButton" name="importBtn">
     <property name="geometry">
      <rect>
       <x>133</x>
       <y>120</y>
       <width>73</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton{color:white;border:1px solid  rgb(46,78,126);border-radius:6px;
background-color: rgb(46,78,126);}
QToolButton:hover{background-color:rgb(66,98,146);}
QToolButton:disabled{background-color:#cecece;}</string>
     </property>
     <property name="text">
      <string>导入</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(190, 190, 190);</string>
     </property>
     <property name="text">
      <string>请 [选择] 或 [输入] 私钥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pathBtn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>73</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{color:white;border:1px solid  rgb(46,78,126);border-radius:6px;
background-color: rgb(46,78,126);}
QToolButton:hover{background-color:rgb(66,98,146);}
QToolButton:disabled{background-color:#cecece;}</string>
     </property>
     <property name="text">
      <string>选择私钥</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>privateKeyLineEdit</tabstop>
  <tabstop>importBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
  <tabstop>pathBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
